src/views/empty-state.component.js

```diff
@@ -151,5 +164,11 @@ angular.module('patternfly.views').component('pfEmptyState', {
         action.actionFn(action);
       }
     };
+
+    ctrl.handleUrlAction = function () {
+      if (ctrl.config.helpLink.urlAction) {
+        ctrl.config.helpLink.urlAction();
+      }
+    };
```

Comment:
This is unnecessary. You can just use ng-click="$ctrl.config.helpLink.urlAction()", if it is not defined nothing will happen, no error.